plugins/org.wso2.developerstudio.eclipse.gmf.esb/src/org/wso2/developerstudio/eclipse/gmf/esb/util/XPathValidatorImpl.java

```diff
@@ -207,9 +212,14 @@ private String getItemListAsString(Iterable<XdmItem> itemList) {
         return outputStr;
     }
 
-    private Iterable<XdmItem> getIterableXpathResult(String xmlString, String xpathExpression) {
+    private Iterable<XdmItem> getIterableXpathResult(String xmlString, String xpathExpression, Map<String, String> namespaces) {
```

Comment:
Shall we limit the character length on a line to 120?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804